cloudflare/data_source_firewall_rule_test.go

```diff
@@ -0,0 +1,122 @@
+package cloudflare
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccCloudflareFirewallRulesMatchPaused(t *testing.T) {
+	t.Parallel()
```

Comment:
You don't want to run these in parallel as they create real resources which if they overlap will taint the expected results.